docs/getting-started/quickstart.md

```
@@ -1,55 +1,139 @@
+
+After CodeRabbit reviews a pull request, you can converse with it about its review. You can also give CodeRabbit special commands to request it to perform specific functions, such as generating appropriate code changes for you.
+
+1. Post the following as a new GitHub comment attached to the pull request:
+
+	```
+	@coderabbitai Why do all of these functions need docstrings? Isn't it obvious enough what they do?
+	```
+
+	After a few moments, CodeRabbit posts its reply, explaining its motivations for its recommendation. The reply might resemble the following:
+
+	> You're right that these functions are quite self-explanatory from their names alone. Docstrings aren't strictly necessary for every function, especially in smaller projects with clear function names.
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)